qemu/tests/cfg/multi_nics_stress.cfg

```diff
@@ -22,5 +22,8 @@
         server_path_win = "c:\\"
         client_path_win = "c:\\"
         test_protocols = "TCP_STREAM TCP_MAERTS UDP_STREAM TCP_RR TCP_CRR UDP_RR"
+        disable_firewall = "netsh firewall set opmode mode=disable"
+        login_timeout = 600
     Linux:
         dhcp_cmd = "for nic in `ls /sys/class/net|grep -v lo`;do arp -a|grep -v $nic && dhclient -r $nic && dhclient -v $nic;done"
+        disable_firewall = "service iptables stop; iptables -F; systemctl stop firewalld.service"
```

Comment:
This should changed in RHEL8, please update accordingly.

Reply:
on rhel8, still could be through systemctl command.
[root@intel-e31225-16-3 ~]# systemctl stop firewalld.service
[root@intel-e31225-16-3 ~]# systemctl status firewalld.service
● firewalld.service - firewalld - dynamic firewall daemon
   Loaded: loaded (/usr/lib/systemd/system/firewalld.service; disabled; vendor preset: enabled)